client/device/ios/create.go

```diff
@@ -149,7 +160,18 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 	// poll until ios config state done
 	client.Logger.Println("waiting for device to reach state done")
 
-	err = retry.Do(ctx, iosconfig.UntilState(ctx, client, deviceCreateOutp.Uid, state.DONE), *retry.NewOptionsWithLogger(client.Logger))
+	err = retry.Do(
+		ctx,
+		iosconfig.UntilState(ctx, client, deviceCreateOutp.Uid, state.DONE),
+		retry.NewOptionsBuilder().
+			Message("waiting for IOS device configuration to be done").
```

Comment:
Waiting for IOS device to be onboarded to CDO...

Reply:
done 👍